website/release_announcement_drafts/v1.3.6.md

```diff
@@ -0,0 +1,19 @@
+We're excited to announce the v1.3.6 release of JBrowse Web! Some highlights of
+this release include:
+
+## Text Searching
+
+Text searching is now available in JBrowse2!
+
+Users will now be able to create their own text indexes with the `jbrowse text-index` command and use them to search for names or identifiers via the location search box of the Linear Genome View.
```

Comment:
Specify JBrowse Web users, since #2296 isn't in yet.

Reply:
I think v1.4.0 makes sense! 👍